packages/braid-design-system/src/lib/components/Tabs/Tabs.tsx

```diff
@@ -134,7 +155,10 @@ export const Tabs = (props: TabsProps) => {
       : selectedIndex;
   const selectedTabButtonEl =
     tabsContext.tabButtonElements[selectedTabIndex.toString()];
-  const activeTab = getActiveTabLinePosition(selectedTabButtonEl);
+
+  useIsomorphicLayoutEffect(() => {
+    setActiveTabPosition(getActiveTabLinePosition(selectedTabButtonEl));
```

Comment:
This is the recommended way: https://react.dev/reference/react/useLayoutEffect#measuring-layout-before-the-browser-repaints-the-screen